assets/armsnippets.jsonc

```diff
@@ -1940,6 +1940,24 @@
         ],
         "description": "Public IP Prefix"
     },
+    "Resource Group": {
+        "prefix": "arm-rg",
+        "body": [
+            "{",
+            "\t\"name\": \"${1:resourceGroup}\",",
+            "\t\"type\": \"Microsoft.Resources/resourceGroups\",",
+            "\t\"apiVersion\": \"2019-10-01\",",
+            "\t\"location\": \"[parameters('location')]\",",

```

Comment:
Consider using a tab stop for location. Because a parameter with the name location may not exist, this would give the consume a out of the box utility for updating the value. The update would look like this:

```
"\t\"location\": \"${2:[parameters('location')]}\",",
```